packages/webapp/src/containers/Consent/index.jsx

```diff
@@ -42,7 +43,6 @@ function ConsentForm({
 
     formState: { errors },
   } = useForm();
-  const [consentVersion] = useState('5.0');
```

Comment:
Ha! What a unique use of `useState`!

Minor, but I don't think the `useState` import is needed anymore if you want to clean that up too.

Reply:
I was scared of changing it to const...😅 
Thank you, I removed the import! ❤️